src/extensions/ExtensionConfig.tsx

```diff
@@ -73,6 +73,15 @@ export const EXTENSIONS: Record<string, Extension> = {
       'Use natural language to generate Cypher queries in NeoDash. Connect to an LLM through an API, and let NeoDash use your database schema + the report types to generate queries automatically. This extension requires APOC Core installed inside Neo4j.',
     link: 'https://neo4j.com/professional-services/',
   },
+  solutionsHive: {
+    name: 'solutionsHive',
+    label: 'Hive Extension',
+    author: 'Neo4j Solutions',
+    image: 'https://ralfvanveen.com/wp-content/uploads/2021/06/Placeholder-_-Begrippenlijst.svg',
```

Comment:
Fix placeholder image.

Reply:
ok

Reply:
Took a screenshot of Hive, uploaded it to the solutions public Google bucket, and replaced the URL